src/api/apiPaths.ts

```diff
@@ -52,3 +52,7 @@ export enum SearchApiPath {
 export enum VerifiedFundingApiPath {
   Nfr = '/verified-funding/nfr',
 }
+
+export enum PersonPreferencesApiPath {
+  PersonPreferences = '/person-preferences/',
```

Comment:
Ser alle andre paths over her ikke har trailing `/`. Har egentlig ikke noe å si, men betyr at vi ikke må legge til `/` før ID når vi skal hente for en bruker da, mens det blir motsatt for alle andre kall